[BUG] Unable to load sqlite3 module

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

During startup of the container, the module for sqlite3 is unable to load.

### Expected Behavior

The container should run without any sqlite3 error messages.

### Steps To Reproduce

Use my docker-compose.yaml and start up container

### Environment

```markdown
- OS: Fedora Server 42
- How docker service was installed: By following the instructions provided on the official docker [website for Fedora](https://docs.docker.com/engine/install/fedora/)
```

### CPU architecture

x86-64

### Docker creation

```bash
thelounge:
    restart: always
    environment:
      TZ: "insert TZ here"
      PUID: "1000"
      PGID: "1001"
    container_name: "thelounge"
    hostname: "thelounge"
    image: "lscr.io/linuxserver/thelounge:latest"
    mac_address: "insert container mac address here"
    networks:
      sol.external:
        ipv4_address: "insert container ipv4 address here"
    ports:
      - "80:80"
      - "443:443"
    volumes:
      - ".thelounge:/config"
```

### Container logs

```bash
[migrations] started
[migrations] no migrations found
───────────────────────────────────────

      ██╗     ███████╗██╗ ██████╗
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝

   Brought to you by linuxserver.io
───────────────────────────────────────

To support LSIO projects visit:
https://www.linuxserver.io/donate/

───────────────────────────────────────
GID/UID
───────────────────────────────────────

User UID:    1000
User GID:    1001
───────────────────────────────────────
Linuxserver.io version: v4.4.3-ls204
Build-date: 2025-07-29T14:30:35+00:00
───────────────────────────────────────
    
[custom-init] No custom files found, skipping...
2025-07-29 14:45:37 [ERROR] Unable to load sqlite3 module. See https://github.com/mapbox/node-sqlite3/wiki/Binaries
2025-07-29 14:45:37 [INFO] The Lounge v4.4.3 (Node.js 22.16.0 on linux x64)
2025-07-29 14:45:37 [INFO] Configuration file: /config/config.js
2025-07-29 14:45:37 [INFO] Available at http://[::]:80/ in private mode
2025-07-29 14:45:37 [INFO] Package thelounge-plugin-shortcuts v1.1.2 loaded
2025-07-29 14:45:37 [INFO] There are packages using the experimental plugin API. Be aware that this API is not yet stable and may change in future The Lounge releases.
2025-07-29 14:45:37 [INFO] Package thelounge-theme-solarized v1.1.9 loaded
2025-07-29 14:45:37 [INFO] User Panda loaded
2025-07-29 14:45:37 [INFO] [thelounge-plugin-shortcuts] [Shortcuts] Loaded 4 shortcuts from /config/packages/thelounge-plugin-shortcuts/shortcuts.json
[migrations] started
[migrations] no migrations found
usermod: no changes
───────────────────────────────────────

      ██╗     ███████╗██╗ ██████╗
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝

   Brought to you by linuxserver.io
───────────────────────────────────────

To support LSIO projects visit:
https://www.linuxserver.io/donate/

───────────────────────────────────────
GID/UID
───────────────────────────────────────

User UID:    1000
User GID:    1001
───────────────────────────────────────
Linuxserver.io version: v4.4.3-ls204
Build-date: 2025-07-29T14:30:35+00:00
───────────────────────────────────────
    
[custom-init] No custom files found, skipping...
2025-07-29 15:47:29 [ERROR] Unable to load sqlite3 module. See https://github.com/mapbox/node-sqlite3/wiki/Binaries
2025-07-29 15:47:29 [INFO] The Lounge v4.4.3 (Node.js 22.16.0 on linux x64)
2025-07-29 15:47:29 [INFO] Configuration file: /config/config.js
2025-07-29 15:47:29 [INFO] Available at http://[::]:80/ in private mode
2025-07-29 15:47:29 [INFO] Package thelounge-plugin-shortcuts v1.1.2 loaded
2025-07-29 15:47:29 [INFO] There are packages using the experimental plugin API. Be aware that this API is not yet stable and may change in future The Lounge releases.
2025-07-29 15:47:29 [INFO] Package thelounge-theme-solarized v1.1.9 loaded
2025-07-29 15:47:29 [INFO] User Panda loaded
2025-07-29 15:47:29 [INFO] [thelounge-plugin-shortcuts] [Shortcuts] Loaded 4 shortcuts from /config/packages/thelounge-plugin-shortcuts/shortcuts.json
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -17,6 +17,7 @@ RUN \
   apk add --no-cache --virtual=build-dependencies \
     build-base \
     git \
+    py3-setuptools \
     python3-dev && \
   echo "**** install runtime packages ****" && \
   apk add --no-cache \

--- a/Dockerfile.aarch64
+++ b/Dockerfile.aarch64
@@ -17,6 +17,7 @@ RUN \
   apk add --no-cache --virtual=build-dependencies \
     build-base \
     git \
+    py3-setuptools \
     python3-dev && \
   echo "**** install runtime packages ****" && \
   apk add --no-cache \

